src/influxdb.c

```diff
@@ -144,6 +170,38 @@ int16_t influxdb_serObject(
     return -1;
 }
 
+int16_t influxdb_serItem(
+    corto_walk_opt *walk,
+    corto_value *info,
+    void *userData)
+{
+    corto_member m = info->is.member.t;
+    corto_type t = corto_value_typeof(info);
+
+    if ((t->kind == CORTO_COMPOSITE) || (t->kind == CORTO_COLLECTION)) {
+        corto_fmt fmt = corto_fmt_lookup("text/json");
+        char* json = (char*)fmt->fromValue(info);
```

Comment:
Memory leak: json string should be deallocated after usage

Reply:
Great Catch.